react/src/demo/demo.css

```diff
@@ -1,17 +1,77 @@
-.grid-stack-item {
-  background-color: rgba(0, 0, 0, 0.125);
-  border: 1px solid rgba(0, 0, 0, 0.125);
-  border-radius: 10px;
-  padding: 10px;
+.btn-primary {
+  color: #fff;
+  background-color: #007bff;
+}
+
+.btn {
+  display: inline-block;
+  padding: .375rem .75rem;
+  line-height: 1.5;
+  border-radius: .25rem;
+}
 
+a {
+  text-decoration: none;
 }
+
+h1 {
+  font-size: 2.5rem;
+  margin-bottom: .5rem;
+}
+
+.sidebar {
+  background: rgb(215, 243, 215);
+  padding: 25px 0;
+  height: 100px;
+  text-align: center;
+}
+.sidebar > .grid-stack-item,
+.sidebar-item {
+  width: 100px;
+  height: 50px;
+  border: 2px dashed green;
+  text-align: center;
+  line-height: 35px;
+  background: rgb(192, 231, 192);
+  cursor: default;
+  display: inline-block;
+}
+
 .grid-stack {
-  background-color: rgba(0, 0, 0, 0.125);
-  border: 1px solid rgba(0, 0, 0, 0.125);
-  border-radius: 10px;
-}
-.grid-stack-placeholder-custom {
-  background-color: rgba(0, 0, 0, 0.125);
-  border: 1px dashed rgba(0, 0, 0, 0.125);
-  border-radius: 10px;
-}
\ No newline at end of file
+  background: #FAFAD2;
+}
+
+.sidebar > .grid-stack-item,
+.grid-stack-item-content {
+  text-align: center;
+  background-color: #18bc9c;
+}
+
+.grid-stack-item-removing {
+  opacity: 0.5;
+}
+.trash {
```

Comment:
this can all be trimmed out... or re-use the main demo file by reference instead...